src/lib/getSchemaUi.ts

```
@@ -0,0 +1,239 @@
+          }
+
+          if (getExamples(schema.items)) {
+            property.subexamples = getExamples(schema.items)
+          }
+
+          if (schema.items.oneOf) {
+            property.meta = { isOneOf: true }
+            property.properties = schema.items.oneOf.map((prop: any) => {
+              const propSchema = { ...prop, type: schema.items.type }
+              return {
+                ...this.schemaToUiProperty('', propSchema),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)